deap/tools/mutation.py

```diff
@@ -205,5 +212,48 @@ def mutESLogNormal(individual, c, indpb):
 
     return individual,
 
+
+######################################
+# GSGP Mutation                      #
+######################################
+
+def semantic_mutation(individual, gen_func=gp.genGrow, pset=None, ms=None, min=2, max=6):
+    """
+    Implementation of the Semantic Mutation operator.
```

Comment:
Extended description + reference + optionally an example.

Reply:
I included equations and a reference to the paper in the docstring, is it okay now?